.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -6,7 +6,7 @@ jobs:
 - job: linux
   pool:
     vmImage: ubuntu-16.04
-  timeoutInMinutes: 360
+  timeoutInMinutes: 600
```

Comment:
Now it worked!

Reply:
Unfortunately, based on the documentation here, it sounds like 360 may still be the max unless the jobs are running on self-hosted agents (instead of Microsoft-hosted ones):
https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#timeouts

Indeed the jobs still timed at at 360 minutes, but I have manually restarted the failing ones just to see if they pass.